components/channel_header/channel_header.js

```diff
@@ -201,8 +201,13 @@ export default class ChannelHeader extends React.PureComponent {
         if (Utils.cmdOrCtrlPressed(e) && e.shiftKey) {
             if (Utils.isKeyPressed(e, Constants.KeyCodes.M)) {
                 e.preventDefault();
+                this.closeQuickSwitchModalIfOpen();
```

Comment:
Is this needed? Based on the ticket we are only concerned when user keys in `ctrl + shift + L`

Reply:
I know, but seemed a similar case, if you feel it isn't I can remove that part

Reply:
Aah, makes sense 👍 . I misinterpreted the shortcut.